sources/Common/DYNSimulationResult.cpp

```diff
@@ -26,7 +26,7 @@ namespace DYNAlgorithms {
 SimulationResult::SimulationResult():
     variation_(-1.),
     success_(false),
-    status_(EXECUTION_PROBLEM_STATUS),
+    status_(NOT_TREATED),
```

Comment:
Are we sure that EXECUTION_PROBLEM_STATUS is always written when the simulation fails? (i.e. no implicit assumption that the default value is EXECUTION_PROBLEM_STATUS)